automated install/basic-install.sh

```diff
@@ -1785,6 +1785,9 @@ create_pihole_user() {
 finalExports() {
     # set or update the variables in the file
 
+    # create the file if it does not exist
+    touch "${setupVars}"
```

Comment:
I wonder if it might be better to introduce a `touch` into `addOrEditKeyValPair` instead? That would reduce the need to type out a `touch` command before calling `addOrEditKeyValPair` on the off chance we create a new file at a later date

Reply:
I thought about this too. I would then also add this to `addKey`